### ISSUE TITLE
New-old issue

### ISSUE THREAD

**Issue by Saxxonknight** [The-Balthazar/BrewLAN#17]
Crash bug still exists, roughly an hour into a game (play time not game time), CTD.  Didn't have any experimentals fielded but several unique BrewLAN units like anti-armor PD, T3 torp launchers, Arty Defense Grid, Field engys.

**Comment by Saxxonknight** [comment]
Got to play tonight with the latest build.  Still having the crash issue, about 30-40 minutes game time.  Was on a custom map vs 9 AI.  I had nothing unique built aside from T1 & T3 Field Engineers, mostly T2 stuff built and just upgrade T3 Facts, Mex & Omni.

Tried loading the last saved game but it crashes not long after.  Suspect problem may be with something the AI is trying to do that I can't see.

**Comment by Saxxonknight** [comment]
I ran the same game on same map using only Waterlag without BrewLAN loaded, was able to build structures on water and did not encounter any crashes during game or loading saved ones late in game.  Trying to narrow down the source of the stoppage.

**Comment by Saxxonknight** [comment]
Got a crash last night and was able to get a crash dump from it.
Related files are here on my google drive, I think I had shared it so you can get there before.

https://drive.google.com/drive/folders/0B7ISwM99CVGUUHNPX3AyUFNiQ2M

**Comment by The-Balthazar** [comment]
The only thing on the log accompanying that appears to be the UI crying about icons it can't find for units that should have icons. Horrendous log spam has caused issues in the past, so it might be worth looking into. Are you running any UI mods?

**Comment by Saxxonknight** [comment]
Yeah, related to Idle Engineers (shows tier, and flags if idle with red number, occasionally it gets a graphics glitch off an engineer and I have to CTLR K it because it won't clear), Mex upgrading (again, one of them it will get a black line glitch that I have to stop and start the feature in the UI which will clear it), Supreme Economy, Notify, Mass Fab Manager.

I have tried testing a few times without the UI mods, but particularly with engineers, it is helpful.  The Mex upgrading helps with efficiency as it automatically starts and pauses a mex to upgrade, so you only have to unpause it, or if something assists it will unpause.
